classes/i18n/interfaces/LocaleInterface.php

```diff
@@ -169,4 +170,11 @@ public function getScripts(?string $locale = null): Scripts;
      * @return array The list of locales with formatted display name
      */
     public function getFormattedDisplayNames(array $filterByLocales = null, array $locales = null, int $langLocaleStatus = LocaleMetadata::LANGUAGE_LOCALE_WITH, bool $omitLocaleCodeInDisplay = true): array;
+
+    /**
+     *  Get UI Translator, which provides all translations used in backend UI (vue.js)
+     *
+     * @return UITranslator provides UITranslator
+    */
+    public function getUiTranslator(): UITranslator;
```

Comment:
When adding methods to this interface, there's another place that needs to be updated (just a workaround to provide hints to the code editor), see:
https://github.com/pkp/pkp-lib/blob/777b10995699b8419a61f62adbceb5cfcba8522d/classes/facades/Locale.php#L44